src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -6848,3 +6848,28 @@ def test_aks_create_node_resource_group(self, resource_group, resource_group_loc
 
         # delete
         self.cmd('aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
+
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_create_with_network_plugin_none(self, resource_group, resource_group_location):
```

Comment:
Queued a [pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=58235096&view=results) to run the test in live mode. Passed.

Please upload the recording file from the artifact of the above pipeline.